README.md

```diff
@@ -11,16 +11,19 @@ A recorder of Ruby programs that produces [CodeTracer](https://github.com/metacr
 you can currently use it directly with
 
 ```bash
-ruby trace.rb <path to ruby file>
-# produces several trace json files in the current directory
-# or in the folder of `$CODETRACER_DB_TRACE_PATH` if such an env var is defined
+ruby trace.rb [--out-dir DIR] <path to ruby file>
+# produces several trace json files in DIR,
+# or in `$CODETRACER_RUBY_RECORDER_OUT_DIR` if DIR is not provided.
```

Comment:
this requires a reform/additional support in `codetracer`

Reply:
OK, make the necessary changes on CodeTracer's side once you bump the version of the ruby-recorder.

Reply:
ok, we can do that. if we decide to directly produce codetracer archives, maybe this can be changed to `_OUT_PATH` but it's not clear yet